udata_gouvfr/templates/c3.html

```diff
@@ -129,7 +129,13 @@ <h3 class="col-xs-12">Les jeux de données remarquables</h3>
                 </li>
                 {% endfor %}
             </ul>
-            <div class="clearfix"></div>
+        </div>
+        <div class="clearfix text-center">
+            <a class="btn btn-default"
+                href="{{ url_for('front.search', badge='c3') }}">
```

Comment:
You should link to `datasets.list` instead of `front.search` as there is only datasets.